mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/json/MessageToElasticSearchJson.java

```diff
@@ -73,9 +73,21 @@ public String convertToJson(MailboxMessage message, List<User> users) throws Jso
         Preconditions.checkNotNull(message);
         switch (indexMessageId) {
             case Required:
-                return mapper.writeValueAsString(IndexableMessageWithMessageId.from(message, users, textExtractor, zoneId, indexAttachments));
+                return mapper.writeValueAsString(IndexableMessage.builder()
```

Comment:
The two lines here were different on previous commit. In second case we should not index messageId, because backend do not support serialization